src/main/java/com/qubit/solution/fenixedu/integration/cgd/services/studentPhoto/CgdStudentPhotoClient.java

```diff
@@ -45,8 +45,11 @@ private MemberCGD(Member member) {
         }
 
         public boolean isStudent() {
+            // The student number being smaller then 999999 is Iscte specific.
```

Comment:
@ajsco this is problematic. The problem was already here but last time I did not noticed it.

The cgd module is part of the fenixedu product so we cannot have specific iscte logic into it.  If both methods for `getPhoto` have the memberCategory can't we use the member category to identify if it's a student, teacher or employee? Also isn't this member code the memberCode? Couldn't we use the CgdMessageUtils.readPersonByMemberCode that uses the correct defined strategy for the institution?

Reply:
Since only Iscte is really using it right now, can this pull request be approved and I will make another one after changing the CgdMessageUtils?

This a blocker bug that we wish to have a release during the next week.

Reply:
@ajsco being a blocking bug and being used only by you guys I'm approving but I'll go talk with you so we can schedule a call this week so we can design a better approach to this